src/lang/english.txt

```diff
@@ -1515,7 +1515,7 @@ STR_CONFIG_SETTING_NEWS_ARRIVAL_FIRST_VEHICLE_OWN               :Arrival of firs
 STR_CONFIG_SETTING_NEWS_ARRIVAL_FIRST_VEHICLE_OWN_HELPTEXT      :Display a newspaper when the first vehicle arrives at a new player's station
 STR_CONFIG_SETTING_NEWS_ARRIVAL_FIRST_VEHICLE_OTHER             :Arrival of first vehicle at competitor's station: {STRING2}
 STR_CONFIG_SETTING_NEWS_ARRIVAL_FIRST_VEHICLE_OTHER_HELPTEXT    :Display a newspaper when the first vehicle arrives at a new competitor's station
-STR_CONFIG_SETTING_NEWS_ACCIDENTS_DISASTERS                     :Accidents / disasters: {STRING2}
+STR_CONFIG_SETTING_NEWS_ACCIDENTS_DISASTERS                     :Accidents/Disasters: {STRING2}
```

Comment:
Capitalizing the second word in these is wrong, IMO.

Also, the spaces do help readability when scanning the settings.

(ditto for equivalent changes below)

Reply:
With this one i have to say: you are wrong, and I am pretty adamant about this one, because this is a rule followed by the large majority of manuals of style in the world, including none other than Chicago. Sorry, but I'm not budging on this one.